firefox_ui_harness/runners/base.py

```diff
@@ -2,7 +2,16 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
-default_prefs = {
+import os
+
+import mozinfo
+from marionette import BaseMarionetteTestRunner
+
+import firefox_ui_tests
+from ..testcase import FirefoxTestCase
+
+
+DEFAULT_PREFS = {
     'app.update.auto': False,
     'app.update.enabled': False,
```

Comment:
At the end I reverted the pref handling code to really use `self.prefs` to set our default preferences. This at least fixes the problem @mjzffr is also seeing for her long run Marionette tests based on our framework. She filed [bug 1146990](https://bugzilla.mozilla.org/show_bug.cgi?id=1146990) for it.